components/card/README.md

```diff
@@ -47,28 +154,11 @@ The `d2l-card` element is a container that provides specific layout using severa
 | `text` | String | Accessible text for the card (will be announced when AT user focuses) |
 
 See the [anchor element docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a) for more information on standard link attributes and their values.
+<!-- docs: end hidden content -->
 
-## d2l-card-footer-link
 
-The `d2l-card-footer-link` element is an icon link that can be placed in the `footer` slot.
-
-```html
-<script type="module">
-  import '@brightspace-ui/core/components/card/card.js';
-  import '@brightspace-ui/core/components/card/card-footer-link.js';
-</script>
-
-<d2l-card>
-  <div slot="content">
-    <div>Hydrology</div>
-  </div>
-  <div slot="footer">
-    <d2l-card-footer-link icon="..." text="..." secondary-text="..." href="..."></d2l-card-footer-link>
-  </div>
-</d2l-card>
-```
-
-**Properties:**
+<!-- docs: start hidden content -->
+### Properties:
```

Comment:
I think these were also supposed to be moved with the rest of the `d2l-card-footer-link` section